test/integration/scheduler_perf/util.go

```diff
@@ -282,3 +308,45 @@ func (tc *throughputCollector) collect() []DataItem {
 
 	return []DataItem{throughputSummary}
 }
+
+// getExtensionPointDurationHist return a labeld Histogram for `scheduler_framework_extension_point_duration_seconds` metric.
+// TODO: Consolidate this method with `testutil.GetHistogramFromGatherer` to remove the duplicated code (#96354).
+func getExtensionPointDurationHist(gatherer metrics.Gatherer) ([]labeledHist, error) {
```

Comment:
why do we need a type, can't we just return a label to histogram map?

Reply:
Done